api/src/test/java/uk/gov/dhsc/htbhf/claimant/service/payments/PaymentServiceTest.java

```diff
@@ -48,32 +54,48 @@
 import static uk.gov.dhsc.htbhf.logging.event.FailureEvent.FAILED_EVENT_KEY;
 import static uk.gov.dhsc.htbhf.logging.event.FailureEvent.FAILURE_DESCRIPTION_KEY;
 
-@ExtendWith(MockitoExtension.class)
+@ExtendWith(SpringExtension.class)
+@SpringBootTest
```

Comment:
Remind me why this needs to be a SpringBootTest? In general we need to reduce the number of SpringBootTests, as our build times are getting dangerously long.

Reply:
So we know that we're successfully creating entities in a valid state before persisting them.